chemie/customprofile/models.py

```diff
@@ -137,6 +137,20 @@ class Profile(models.Model):
     voted = models.BooleanField(default=False)
     eligible_for_voting = models.BooleanField(default=False)
     balance = models.DecimalField(max_digits=6, decimal_places=2, default=0)
+    
+    coffee_subscription = models.BooleanField(
+        default=True, verbose_name="Aboner på kaffe pushvarsler"
+    )
+    
+    news_subscription = models.BooleanField(
+        default=True, verbose_name="Aboner på nyheter pushvarsler"
+    )
+
+    happyhour_subscription = models.BooleanField(
+        default=True, verbose_name="Aboner på happy hour pushvarsler"
+    )
```

Comment:
Inflexible to more types of subscription and has a bloating effect on the Profile model. Consider replacing by a ManyToManyField